lib/src/compile.ts

```diff
@@ -10,7 +10,12 @@ import {PacketTransformer} from './embedded-compiler/packet-transformer';
 import {MessageTransformer} from './embedded-protocol/message-transformer';
 import {Dispatcher} from './embedded-protocol/dispatcher';
 import {deprotifyException} from './embedded-protocol/utils';
-import {InboundMessage} from './vendor/embedded-protocol/embedded_sass_pb';
+import {
+  InboundMessage,
+  Syntax as SyntaxMap,
+} from './vendor/embedded-protocol/embedded_sass_pb';
```

Comment:
I would suggest `import * as proto` and using the namespaced `proto.Syntax` to avoid arbitrarily renaming this import.